tests/ut-ncnnapi.cc

```diff
@@ -299,3 +300,33 @@ TEST(ncnnapi, service_lstm)
   rmdir(csvts_repo.c_str());
 }
 #endif
+
+TEST(ncnnapi, ocr)
+{
+  // create service
+  JsonAPI japi;
+  std::string sname = "ocr";
+  std::string jstr
+      = "{\"mllib\":\"ncnn\",\"description\":\"ocr\",\"type\":"
+        "\"supervised\",\"model\":{\"repository\":\""
+        + ocr_repo
+        + "\"},\"parameters\":{\"input\":{\"connector\":\"image\",\"ctc\":"
+          "true, \"height\":136,\"width\":220},\"mllib\":{\"nclasses\":69}}}";
+  std::string joutstr = japi.jrender(japi.service_create(sname, jstr));
+  ASSERT_EQ(created_str, joutstr);
+
+  // predict
+  std::string jpredictstr = "{\"service\":\"ocr\",\"parameters\":{\"input\":{}"
+                            ",\"output\":{\"confidence_threshold\":0,\"ctc\":"
+                            "true,\"blank_label\":0}},\"data\":[\""
+                            + ocr_repo + "word_ocr.jpg\"]}";
+  joutstr = japi.jrender(japi.service_predict(jpredictstr));
+  JDoc jd;
+  std::cout << "joutstr=" << joutstr << std::endl;
+  jd.Parse<rapidjson::kParseNanAndInfFlag>(joutstr.c_str());
+  ASSERT_TRUE(!jd.HasParseError());
+  ASSERT_EQ(200, jd["status"]["code"]);
+  ASSERT_TRUE(jd["body"]["predictions"].IsArray());
+  ASSERT_TRUE(jd["body"]["predictions"].Size() == 1);
+  ASSERT_TRUE(jd["body"]["predictions"][0]["classes"][0]["cat"] == "beleved");
```

Comment:
beleved ? :)

Reply:
yup, using previous ncnn (ie our old fork) I also get "beleved", while using caffe I get "believed"

Reply:
Let's try with fp32 activated (fp16 is default), would be surprised it corrects the output though...